### PR TITLE
chore: auth saga test

### PR DESCRIPTION
**Pull request by mychalvlcek**
test for auth/sagas - authorize

### CODE REVIEW

File: tests/modules/auth.sagas.spec.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import test from 'tape';
+
+import { call, put, select } from 'redux-saga/effects';
+import {getApiService, getUser, getAuth} from '../../src/modules/auth/selectors';
+import { sagas, actions } from '../../src/modules/auth';
+
+var apiMock = {
```
Comment (mychalvlcek) [inline tests/modules/auth.sagas.spec.js:7]: mock object

File: tests/modules/auth.sagas.spec.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+test('modules/ath/sagas:authorize test - success', (t) => {
+
+	const saga = sagas.authorize({}, {});
+
+	t.deepEqual(saga.next().value, select(getApiService), 'authorize Saga must select ApiService');
+	t.deepEqual(saga.next(apiMock).value, call(apiMock.login, {}, {}), 'authorize Saga must call ApiService.login');
+
+	var loginResult = {user: {}};
```
Comment (mychalvlcek) [inline tests/modules/auth.sagas.spec.js:21]: mock data - result from api call

**Comment by mychalvlcek** [comment]
covered: authorize, logout saga
missing: authFlow saga

File: tests/modules/auth.sagas.spec.js (hunk 1 above)
Comment (brabeji) [inline tests/modules/auth.sagas.spec.js:7]: there is mock api service in `src/mocks`, don't you want to use that?

File: tests/modules/auth.sagas.spec.js (hunk 2 above)
Comment (brabeji) [inline tests/modules/auth.sagas.spec.js:21]: typo on line 12

Comment (mychalvlcek) [inline tests/modules/auth.sagas.spec.js:21]: updated all occurrences

**Pull request merged**